### ISSUE TITLE
Issue with k8s.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/

### ISSUE THREAD

**Issue by hopik007** [kubernetes/website#27381]
" If the command returns a non-zero value, the kubelet kills the container and restarts it."

Mean k8s sent sigterm or sigkill ?
It is important to know. 

Thanks you

**Comment by sftim** [comment]
/kind feature

@hopik007  it might be a while before we can document this. I think the truth is that the kubelet delegates container shutdown to the [container runtime](https://kubernetes.io/docs/setup/production-environment/container-runtimes/). On Windows nodes, the mechanism for forcing container shutdown doesn't use POSIX signals at all.

I think it is worth clarifying that the kubelet does not directly and literally send a KILL signal to a container. However, it's a low priority improvement.

/priority backlog
/triage accepted
/lanagage en
